organizationinvitation/client.go

```diff
@@ -0,0 +1,53 @@
+// Package organizationinvitation provides the Organization Invitations API.
+package organizationinvitation
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/clerk/clerk-sdk-go/v2"
+)
+
+//go:generate go run ../cmd/gen/main.go
+
+const path = "/organizations"
+
+// Client is used to invoke the Organization Invitations API.
+type Client struct {
+	Backend clerk.Backend
+}
+
+type ClientConfig struct {
+	clerk.BackendConfig
+}
+
+func NewClient(config *ClientConfig) *Client {
+	return &Client{
+		Backend: clerk.NewBackend(&config.BackendConfig),
+	}
+}
+
+type CreateParams struct {
+	clerk.APIParams
+	EmailAddress    *string          `json:"email_address,omitempty"`
+	Role            *string          `json:"role,omitempty"`
+	RedirectURL     *string          `json:"redirect_url,omitempty"`
+	InviterUserID   *string          `json:"inviter_user_id,omitempty"`
+	PublicMetadata  *json.RawMessage `json:"public_metadata,omitempty"`
```

Comment:
🔧 I don't think we should be using pointers to [`json.RawMessage`](https://pkg.go.dev/encoding/json#RawMessage) since it's an alias for `[]byte`. It's actually slightly problematic to pass pointers to slices as they can suffer unintended mutations.